lib/pure/stats.nim

```diff
@@ -338,31 +338,30 @@ runnableExamples:
 
 
 when isMainModule:
-  proc clean(x: float): float =
-    result = round(1.0e8*x).float * 1.0e-8
+  proc `=~`(x, y: float): bool = (abs(x-y) < 1e-8)
```

Comment:
not a great definition for approximate equality since it makes `1e-12 =~ 0.0`

See https://github.com/nim-lang/RFCs/issues/275 for better options